pkg/oobpolicy/apis/aci.oob/v1/types.go

```diff
@@ -0,0 +1,38 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +kubebuilder:validation:Enum=Immediate;OnDemand
+type VmmEpgDeploymentImmediacyType string
+
+const (
+	VmmEpgDeploymentImmediacyTypeImmediate VmmEpgDeploymentImmediacyType = "Immediate"
+	VmmEpgDeploymentImmediacyTypeOnDemand  VmmEpgDeploymentImmediacyType = "OnDemand"
+)
+
+type OutOfBandPolicySpec struct {
+	TunnelEpAdvertisementInterval int                           `json:"TunnelEpAdvertisementInterval,omitempty"`
```

Comment:
needs to be uint64_t

Reply:
Also split this into two structs with embedding, so you can reuse in the rest of the code.